README.md

```diff
@@ -106,9 +106,8 @@ For a brief introduction to configuring the syslog-ng application, see the [quic
 
 ### Sponsors
 
-[Balabit](http://www.balabit.com/) is the original creator and largest current
-sponsor of the syslog-ng project. They offer support,
-professional services, and addons you may be interested in
+- [Axoflow](https://axoflow.com) is the company of Balazs Scheidler, the original creator and main developer of syslog-ng, and the creators of the Kubernetes Logging Operator. Currently Axoflow is the most active contributor of syslog-ng, and offers commercial support, professional services, and related products.
+- [Balabit](http://www.balabit.com/) is the original creator of the syslog-ng project. They offer support, professional services, and addons you may be interested in.
```

Comment:
Balabit does not exist anymore.  Therefore, they cannot offer "support, PS, etc.".